### PR TITLE
Some cleanups

### PR DESCRIPTION
**Pull request by fkorotkov**
While checking sources for the first time I found a few minor issues.

### CODE REVIEW

**Comment by fkorotkov** [comment]
@facebook-github-bot signed.

File: scripts/autofix_lint.py [hunk 1]
```diff
@@ -277,8 +277,7 @@ def translateFunc(problem):
                 start_index = translate_metadata['start_index']
                 end_index = translate_metadata['end_index']
                 offset_array = translate_metadata['offset_array']
-                if (problem.line >= start_index and
-                        problem.line < end_index):
+                if start_index <= problem.line < end_index:
```
Comment (sdwilsh) [inline scripts/autofix_lint.py]: Today I learned this was valid in Python.  Fancy.

**Comment by sdwilsh** [comment]
I'm just curious if you used a tool to spot these or came across them by hand?

**Comment by fkorotkov** [comment]
I was checking source in IntelliJ and saw some warning in the editor like one you mentioned above. And then I ran additional inspections on all sources and picked a few obvious fixes. There were way more warnings and possible bugs but I don't have so much context on Buck right now to evaluate them.

**Pull request closed without merging**